src/styles/components/_card.scss

```diff
@@ -0,0 +1,82 @@
+.card {
+  width: 20rem;
+  margin: auto;
+  padding: 3.2rem 1.6rem 1.6rem;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+
+  @include border(0.1rem, $color-card-bgc, 0.5rem);
+
+  &__image {
+    width: 16rem;
+    height: 13.4rem;
+    background-image: url('../images/imac.jpeg');
+    background-size: cover;
+    margin-bottom: 4rem;
+
+    @extend %bgc-image;
+  }
+
+  &__name,
+  &__reviews,
+  &__price.value {
```

Comment:
There seems to be a typo in the class name. The class should be `&__price` instead of `&__prices` as per the BEM naming convention. Please correct this to match the singular form as used elsewhere in your code.